nakama/main.go

```diff
@@ -277,7 +278,7 @@ func initCardinalEndpoints(logger runtime.Logger, initializer runtime.Initialize
 	if err != nil {
 		return fmt.Errorf("failed to get list of cardinal endpoints: %w", err)
 	}
-
+	matchQueryRoutes := regexp.MustCompile("^query/*")
```

Comment:
cardinalListAllEndpoints gets the list of tx endpoints as well as the list of read endpoints, and combines them into a single slice. Since it's so easy to get the two separate lists of endpoints, I think it would be cleaner to remove the regex code here, and split this initCardinalEndpoints into two functions: initCardinalReadEndpoints and initCardinalTxEndpoints.